MySaver/Views/UserDetailsPage.xaml

```diff
@@ -31,16 +27,12 @@
                     <ColumnDefinition Width="0.5*"/>
                     <ColumnDefinition Width="0.5*"/>
                 </Grid.ColumnDefinitions>
-                <Label VerticalOptions="Start"
-                       Margin="10">
+                <Label VerticalOptions="Start" Margin="10">
                     <Label.FormattedText>
                         <FormattedString>
-                            <Span Text="Gender"
-                                  TextColor="Black"/>
-                            <Span Text="&#x0a;"
-                                  TextColor="Black"/>
-                            <Span Text="Male"
-                                  TextColor="Black"/>
+                            <Span Text="Gender" TextColor="Gray"/>
+                            <Span Text="&#x0a;" TextColor="Gray"/>
+                            <Span Text="Male" TextColor="Gray"/>
```

Comment:
The text color here is explicitly gray, which overrides the light/dark theme settings, I think it can be a bit difficult to read gray text when using the dark theme so consider removing "TextColor="Gray" from label text. (also in the user summary page maybe)

Reply:
Changed it, although switching between light/dark themes requires to leave and come back to the same page to see the results. Gonna work on that a bit later